src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -52,7 +52,9 @@
                 Inherited Packages (#{@ipackages.length})
       .tab-content#packages-tabs-content
         .tab-pane.fade.show.active#packages{ role: 'tabpanel', aria: { labelledby: 'packages-tab' } }
-          = render partial: 'project_packages', locals: { project: @project, packages: @packages }
+          = render partial: 'project_packages', locals: { project: @project, packages: @packages, remote_projects: @remote_projects,
+                                                          package_name: @package_name, package_title: @package_title,
```

Comment:
Are `@package_name`, `@package_title` and `@package_description` needed? I believe they are always `nil` as they aren't defined in the project controller.

Reply:
@dmarcoux you are right, but this method  https://github.com/openSUSE/open-build-service/blob/ac88023dad87a4badbd0e713df62dd78b5b0608f/src/api/app/controllers/webui/package_controller.rb#L507 expects ```@package_name```. But in the ```check_package_name_for_new``` the param of this form gets assigned to ```@package_name``` so it would work with out it. Hope i got it right :D. For sure i can remove, ```@package_title``` and ```@package_description```. What do you think about the ```@package_name```?

Reply:
The value `@package_name` is assigned in the modal's form [here](https://github.com/openSUSE/open-build-service/pull/6458/files#diff-7aed9af7b6037de41030165c766541fcR14), but since it's `nil`, it doesn't set anything in the text field. So when you submit the form, there's a [`before_action` for `save_new`](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/package_controller.rb#L46) which is triggered and this is where [`@package_name` is set](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/package_controller.rb#L527) from the data the form submitted.

Reply:
@dmarcoux changed it :)